raiden/transfer/mediated_transfer/target.py

```diff
@@ -255,6 +289,19 @@ def handle_block(target_state, channel_state, block_number):
     return iteration
 
 
+def handle_lock_expired(
+        target_state: TargetTransferState,
+        state_change: ReceiveLockExpired,
+        channel_state: NettingChannelState,
+):
+    result = channel.handle_receive_lock_expired(channel_state, state_change)
+
+    return TransitionResult(
+        None,
```

Comment:
this is not correct, the target should not be unconditionally cleaned, it should only be cleaned **iff** the message is valid.